server/src/routers/conversation.router.ts

```diff
@@ -41,6 +45,38 @@ conversationRouter.get(
   }),
 );
 
+conversationRouter.get(
+  '/all',
+  authMiddleware,
+  requestHandlerMiddleware(async (req, res) => {
+    const currentUser = getUserFromRequestContext(req);
+    const groupName = (req.query.groupName as string) || '';
+    const participantName = (req.query.participantName as string) || '';
```

Comment:
how do you plan to differentiate the participantName from the groupName when sending the request?
In the frontend we have a simple search input which only works with a single string -> it can set only 1 query param. 
What I suggest is to have a single search query param and first filter convos by groupName and then append the resulting convos from filtering the participants names
If the search = 'Ivan' first show the group 'Rojden den na Ivan' and then the private/group chats that have Ivan in them

Reply:
ah yes true, we have only a single search query.

the approach you suggest is alright and we can do that.

if we need more fine tuning another option would be adding filters to the search query such as "user=Ivan" so the whole search string would look something like "rojden den user:ivan" or so, which would parsed on the backend to give us the appropriate query tokens.

let me know

Reply:
Make it accept only one query parameter and pass it to both group names and participants' names.